nvidia-update.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I write my internal/development scripts like this because they target Linux as well as macOS. However, for macOS I believe this is incorrect.

`/bin/sh` is not user configurable on macOS. If the user changes this they're likely in a whole world of hurt.

Reply:
Agree.